editor/store/selectors.js

```diff
@@ -563,6 +563,45 @@ export const getBlocks = createSelector(
 	]
 );
 
+const unfoldBlockIds = ( state, block ) => {
+	const clientIds = [];
+	if ( block.name === 'core/block' ) {
+		clientIds.push( get( getSharedBlock( state, block.attributes.ref ), [ 'clientId' ], null ) );
```

Comment:
I think we may need to apply an unfoldOperation to the result of getSharedBlock because that block may have other blocks nested inside.